### PR TITLE
MXP-86 Addressed box-sizing issue due to padding being taken into con…

### PR DESCRIPTION
**Pull request by jdashley27**
Needed to address an issue where some code bases were ignoring padding to be considered with the width, whereas the MDL storybook was changing the box size based upon the set padding.

### CODE REVIEW

File: components/menu/OverflowMenu/style.module.scss [hunk 1]
```diff
@@ -29,13 +29,16 @@
...
+  width: 282px;
   display: inline-block;
   position: absolute;
-  top: 100%;
-  left: 0;
+  top: 60%;
+  left: 60%;
   box-shadow: 0 12px 15px 0 rgba(0, 0, 0, 0.14);
   z-index: 9;
+  -webkit-box-sizing: border-box; /* Safari/Chrome, other WebKit */
```
Comment (tarikhamilton) [inline components/menu/OverflowMenu/style.module.scss]: You don't need vendor prefixes. All CSS runs through autoprefixer.

Comment (jdashley27) [reply components/menu/OverflowMenu/style.module.scss]: Works for me, thanks

**Comment by jdashley27** [comment]
@simonwebdesign 
I've updated the styles, so it should fall in line with the links you had shared with me.

@tarikhamilton 
Please look over the updated PR, as I removed the vendor prefixes, and added some slight functionality/unit tests to make it fall in line with the links Simon had included.

File: components/menu/OverflowMenu/style.module.scss [hunk 2]
```diff
@@ -6,36 +6,47 @@
...
+  width: 36.111111111%;
+  height: 44.444444444%;
+}
+
+.overflow-menu__svg {
+  fill: $color-deepgrey;
+}
+
+.overflow-menu__svg--opened {
+  fill: $color-m-primary-blue;
```
Comment (tarikhamilton) [inline components/menu/OverflowMenu/style.module.scss]: Legacy color variable

File: components/menu/OverflowMenu/style.module.scss [hunk 3]
```diff
@@ -6,36 +6,47 @@
...
 }
 
-.overflow-menu__svg {
+.overflow-menu__svg,
+.overflow-menu__svg--opened {
   @include completely-center();
-  width: 40%;
-  height: 60%;
+  width: 36.111111111%;
+  height: 44.444444444%;
```
Comment (tarikhamilton) [inline components/menu/OverflowMenu/style.module.scss]: These look like magic numbers. What are they for?

File: components/menu/OverflowMenu/style.module.scss [hunk 4]
```diff
@@ -6,36 +6,47 @@
...
+  width: 282px;
   display: inline-block;
   position: absolute;
-  top: 100%;
-  left: 0;
+  top: 60%;
+  left: 60%;
   box-shadow: 0 12px 15px 0 rgba(0, 0, 0, 0.14);
   z-index: 9;
+  box-sizing: border-box; /* Addresses padding affecting box-width */
```
Comment (tarikhamilton) [inline components/menu/OverflowMenu/style.module.scss]: Remove comment?

File: components/menu/OverflowMenu/style.module.scss [hunk 5]
```diff
@@ -6,36 +6,47 @@
 }
 
 .overflow-menu__button {
-  @include circle(26px);
+  @include circle(36px);
   padding: $size-md;
```
Comment (tarikhamilton) [inline components/menu/OverflowMenu/style.module.scss]: Legacy size value used

File: components/menu/OverflowMenu/style.module.scss (hunk 2 above)
Comment (jdashley27) [reply components/menu/OverflowMenu/style.module.scss]: Is there a different one I should be using? I do see a different one of $color-blue.

File: components/menu/OverflowMenu/style.module.scss (hunk 3 above)
Comment (jdashley27) [reply components/menu/OverflowMenu/style.module.scss]: They control the ellipsis size. This allows them to scale relatively with the overall overflow button. I could hard code them to be 13px by 16px, but if the circle gets smaller or bigger, it'll lock the ellipsis size.

File: components/menu/OverflowMenu/style.module.scss (hunk 5 above)
Comment (jdashley27) [reply components/menu/OverflowMenu/style.module.scss]: Any idea as to what the size should be? From my inspect on zeroheight, it shows as 36px. Link: https://zeroheight.com/73kwjxh8i/p/52140/i/861670

File: components/menu/OverflowMenu/style.module.scss (hunk 4 above)
Comment (jdashley27) [reply components/menu/OverflowMenu/style.module.scss]: I removed the comment.

**Pull request merged**